DEBUG - 2025-09-11 05:23:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-11 05:23:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-11 05:23:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-11 05:23:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-11 05:31:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-11 05:31:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-11 16:30:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-11 16:30:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-09-11 16:31:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-09-11 16:31:11 --> CSRF token verified.
ERROR - 2025-09-11 16:31:11 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'lms_pecayo' in C:\xampp\htdocs\ITE311-PECAYO\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\ITE311-PECAYO\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'lms_pecayo', 3306, '', 0)
#1 C:\xampp\htdocs\ITE311-PECAYO\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ITE311-PECAYO\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ITE311-PECAYO\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ITE311-PECAYO\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ITE311-PECAYO\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\ITE311-PECAYO\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\ITE311-PECAYO\system\CodeIgniter.php(933): App\Controllers\Auth->attemptLogin()
#8 C:\xampp\htdocs\ITE311-PECAYO\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\ITE311-PECAYO\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-PECAYO\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-PECAYO\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-PECAYO\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'lms_pecayo' in C:\xampp\htdocs\ITE311-PECAYO\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\ITE311-PECAYO\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ITE311-PECAYO\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ITE311-PECAYO\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\ITE311-PECAYO\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\ITE311-PECAYO\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\ITE311-PECAYO\app\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\ITE311-PECAYO\system\CodeIgniter.php(933): App\Controllers\Auth->attemptLogin()
#7 C:\xampp\htdocs\ITE311-PECAYO\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp\htdocs\ITE311-PECAYO\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ITE311-PECAYO\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ITE311-PECAYO\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\ITE311-PECAYO\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-11 16:31:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'lms_pecayo'
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(32): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->attemptLogin()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-11 16:31:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
